.github/workflows/build-test.yaml

```diff
@@ -0,0 +1,68 @@
+name: Build and Test
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    shell: bash
+
+env: 
+  HOMEBREW_NO_AUTO_UPDATE: 1
+
+jobs:
+
+  build:
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-18.04]
+        python-version: [3.8]
+        minimal: [false]
```

Comment:
It looks to me like the variable value shows up in the GHA workflow name.  Would it be possible to instead of having `minimal: [true, false]` have something like `extent: [minimal, full]`?

Reply:
done